packages/expo/package.json

```diff
@@ -84,6 +84,8 @@
     "expo-gl-cpp": "~2.0.0",
     "expo-google-sign-in": "~2.0.0",
     "expo-image-loader-interface": "~2.0.0",
+    "expo-keep-awake": "*",
```

Comment:
should I publish these before landing the PR?

Reply:
No, you don't have to. We will publish it together with `expo` package during next release. We also used to use `~1.0.0` instead of `*` as the initial version (even if not published yet).